### DISCUSSION TITLE
Suggestions for Service Design section in the playbook

### DISCUSSION THREAD

**Discussion by anabrighenti** [nhsbsa/nhsbsa-digital-playbook#244] [UCD]
We propose that the following guidance be included in the playbook to guide what actions and artefacts are expected in service design and will support assessments at different project phases: 

 
**Discovery:** 

- Actions: Empathy mapping, ecosystem mapping, assumption mapping, problem framing. 
- Artefacts: All of the above 

**Alpha:** 

- Actions: Service blueprint, including mapping user actions, map employees’ frontstage and backstage actions, map support processes and evidence. User journey maps. 
- Artefacts: Service blueprint, user journey maps. 

**Private Beta:** 

- Actions: Iterate on previous phase artefacts 
- Artefacts: All the above 

**Public Beta:** 

- Actions: Iterate on previous phase artefacts 
- Artefacts: All the above

**Comment by nhsbsa-sthil** [comment]
@anabrighenti Would you expect this to sit on the current pages i.e.

* [Service design at Discovery](https://nhsbsa.github.io/nhsbsa-digital-playbook/design/service-designer/discovery/)
* [Service design at Alpha](https://nhsbsa.github.io/nhsbsa-digital-playbook/design/service-designer/alpha/)
* [Service design at Beta and Live](https://nhsbsa.github.io/nhsbsa-digital-playbook/design/service-designer/beta-live/)

Also, is the above, the content you'd want on there?

**Reply by NettySP8** [reply]
Yeah Stephen thats the idea as we feel that supports whats expected in the role but easier broken down in the assessment phases

**Reply by nhsbsa-sthil** [reply]
Sounds good! 💯

**Reply by nhsbsa-sthil** [reply]
We could stick to your content exactly, or perhaps you could lay it out something like this....?

![image](https://github.com/nhsbsa/nhsbsa-digital-playbook/assets/45825845/39fb2819-f667-422c-974d-c2018c243bfe)

I'll let you decide amongst yourselves what you want the content to be but perhaps if it gets agreed, we can discuss actioning this at the next working group meeting (16/11/2023)?

**Reply by NettySP8** [reply]
Yeah that looks really good but maybe the content sway more towards what would be expected to support a service assessment too as that looks like a suggestion and if they decide to ignore the suggestion theyd be in a sticky situation come assessment?

**Reply by nhsbsa-sthil** [reply]
I'm happy with whatever you folks think is needed to be honest 👍

**Reply by NettySP8** [reply]
We'll have a look, thanks Stephen :)

**Reply by nhsbsa-sthil** [reply]
@NettySP8 Think I suggested in the last playbook working group meeting that this could be an item where we could start to add the above content into the playbook but would you like me to hold off for now until you've discussed what Service Designers would be expected to contribute to a service assessment - or do you want to pick that up as a separate bit of work?

**Reply by NettySP8** [reply]
Hey Stephen we are happy to go ahead after what we said at the working group and have it added if you are?

**Reply by nhsbsa-sthil** [reply]
Sounds good to me. I'll get the ball rolling.
